src/util/load-async.js

```diff
@@ -119,6 +119,18 @@ const loaders = new Map()
     /* webpackChunkName: "component-form-submissions" */
     '../components/form/submissions.vue'
   )))
+  .set('FormSubmission', loader(() => import(
+    /* webpackChunkName: "component-form-submission" */
+    '../components/form/submission.vue'
+  )))
+  .set('EnketoIframe', loader(() => import(
+    /* webpackChunkName: "component-enketo-iframe" */
+    '../components/enketo-iframe.vue'
+  )))
+  .set('WebFormRendered', loader(() => import(
+    /* webpackChunkName: "component-web-form-renderer" */
+    '../components/web-form-renderer.vue'
+  )))
```

Comment:
I don't see these two components being loaded async anywhere. `WebFormRendered` looks misspelled here, so I doubt this key is being utilized. `FormSubmission` will be loaded async, since it's a route component and `asyncRoute()` is used, but the components that `FormSubmission` uses don't also need to be async.